Resources/Textures/_NF/Objects/Weapons/Guns/Ammunition/Casings/410shells.rsi/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "https://github.com/discordia-space/CEV-Eris/raw/8a848cde3e8610455f77704e861e5a06315a3b0e/icons/obj/ammo.dmi",
+    "states": [
+        {
+            "name": "base"
```

Comment:
I'm struggling to see any difference between regular shotgun shell sprite and 410 shell sprite

Reply:
One is smaller as it is IRL